src/pages/background/index.js

```diff
@@ -43,7 +43,7 @@ chrome.runtime.onMessage.addListener(function (message, sender) {
   const { tab } = sender
 
   console.groupCollapsed(`RECEIVE: ${type}`)
-  console.log(message)
+  console.log(payload)
```

Comment:
We only ever pass the payload, so that felt like the important info to highlight.